### PR TITLE
Try to fix tests syncer

### PR DESCRIPTION
**Pull request by jonathanmetzman**
The existing setup doesn't terminate. Probably because there's over 157k testcases.

### CODE REVIEW

File: src/python/other-bots/chromium-tests-syncer/run.py [hunk 1]
```diff
@@ -37,14 +37,20 @@
...
 
 
 def unpack_crash_testcases(crash_testcases_directory):
   """Unpacks the old crash testcases in the provided directory."""
+  count = 0
   for testcase in ndb_utils.get_all_from_model(data_types.Testcase):
+    count += 1
+    if count >= MAX_TESTCASES:
+      logs.info(f'{MAX_TESTCASES} testcases reached.')
+      break
```
Comment (vitorguidi) [inline src/python/other-bots/chromium-tests-syncer/run.py:53]: What is the idea here?  This script will run several times and consume the old test cases slowly, or is this a hard cap untill some sort of corpus pruning task runs?

**Review by vitorguidi — APPROVED** [review]
lgtm

File: src/python/other-bots/chromium-tests-syncer/run.py (hunk 1 above)
Comment (jonathanmetzman) [reply src/python/other-bots/chromium-tests-syncer/run.py:53]: Hard cap.

Comment (jonathanmetzman) [reply src/python/other-bots/chromium-tests-syncer/run.py:53]: There's never any pruning for here. This source has basically been polluted over the years and is unusable now.

Comment (jonathanmetzman) [reply src/python/other-bots/chromium-tests-syncer/run.py:53]: Maybe I will manually prune it, though I don't know how much it will move the needle.

Comment (mi-ac) [reply src/python/other-bots/chromium-tests-syncer/run.py:53]: It'd be great if we could get the latest 500 though...

**Pull request merged**